layouts/shortcodes/login-form.html

```diff
@@ -1,5 +1,5 @@
 <div id="loginForm"></div>
 {{ $env := cond .Site.IsServer "\"development\"" "\"production\"" }}
 {{ $opts := dict "defines" (dict "process.env.NODE_ENV" $env) }}
-{{ $js := resources.Get "js/login.ts" | babel | js.Build $opts }}
+{{ $js := resources.Get "js/login.tsx" | babel | js.Build $opts }}
```

Comment:
every scripts under js, which will be used in the shortcodes is planned to be `ts` not `tsx`.